tools/unicode_properties_parse/grapheme_break_property_data_gen.py

```diff
@@ -76,6 +76,14 @@ def compact_property_ranges(input: list[PropertyRange]) -> list[PropertyRange]:
 }};
 """
 
+INTERVALS_TEMPLATE = """
+{filename}
+{timestamp}
+inline constexpr char32_t _{prop_name}_ranges[{size}] = {{
+    {data}
+}};
+"""
+

```

Comment:
`__printable_ranges` needs at least 16 bits to hold the size. The largest range is `20000..2A6DF` whose size is 42720.

I'll see how to modify `_Unicode_property_data` to allow this.

Reply:
That actually doesn't matter (although since the property only has one value it's fine to allow this), you just split the range into one range of 2^12 and one smaller range containing the remainder.